$NetBSD: patch-af,v 1.3 2012/03/05 04:44:17 dholland Exp $

- needs <time.h> on linux
- avoid <sys/errno.h>, not standard and fails on qnx
- dragonfly support

--- syshdr.h.orig	1998-11-16 17:03:49.000000000 +0000
+++ syshdr.h
@@ -15,8 +15,8 @@
 #include <sys/socket.h>
 #include <unistd.h>
 #include <sys/time.h>
+#include <time.h>
 #include <fcntl.h>
-#include <sys/errno.h>
 #include <errno.h>
 #include <sys/param.h>
 #include <sys/stat.h>
@@ -47,6 +47,9 @@
 #		define BSD4_4_LITE2
 #	endif
 #endif
+#if defined(__DragonFly__)
+#define BSD4_4_LITE2
+#endif
 #if defined(BSD4_4) && !defined(BSD4_4_LITE2)
 #	define BSD4_4_LITE
 #endif
